subscribe_test.go

```diff
@@ -259,6 +262,61 @@ func TestSubscribe(t *testing.T) {
 	testSubscribe(t, 3)
 }
 
+func testSubscribeLogs(t *testing.T, hub *Hub) {
+	t.Helper()
+	s, _ := hub.transport.(*LocalTransport)
+
+	go func() {
+		for {
+			s.RLock()
+			empty := s.subscribers.Len() == 0
+			s.RUnlock()
+
+			if empty {
+				continue
+			}
+
+			hub.transport.Dispatch(&Update{
```

Comment:
Dispatching updates isn't necessary for this test as you log at subscription time.